tests/unit/config/config_test.py

```diff
@@ -1138,7 +1138,21 @@ def test_expand_path_with_tilde(self):
         self.assertEqual(result, user_path + 'otherdir/somefile')
 
 
-@pytest.mark.xfail(IS_WINDOWS_PLATFORM, reason='paths use slash')
```

Comment:
The test failures are because this decorator was removed from `BuildPathTest`.

I don't think we should merge with a red build. Please re-add this `xfail` so the build is green.